src/components/TransactionList/TransactionListTs.ts

```diff
@@ -259,11 +295,47 @@ export class TransactionListTs extends Vue {
    * Hook called at each page change
    */
   public onPageChange(page: number): void {
-    if (page > this.countPages) page = this.countPages
-    else if (page < 1) page = 1
+    if (page === this.countPages) {
+      this.loadMore()
+    } else if (page > this.countPages) {
+      page = this.countPages
+    } else if (page < 1) {
+      page = 1
+    }
     this.currentPage = page
   }
 
+  /**
+   * Loads next page of transactions from repository
+   */
+  public loadMore() {
+    if (!this.currentConfirmedPage.isLastPage) {
+      this.$store.dispatch('transaction/LOAD_TRANSACTIONS', {
+        pageNumber: ++this.currentConfirmedPage.pageNumber,
+        pageSize: 20,
```

Comment:
should the pageSize be hardcoded at all?

Reply:
There are two pagination options implemented (pagination-type: (conventional) pagination | (infinite)scroll). The developer can set it through component properties while reusing the same component, the default type is 'scroll'. To support them both and to comply with the current logic I decided to use repository level pagination instead of pagestreamer(although I liked the streamer idea very much). Hope that makes sense.

Reply:
> should the pageSize be hardcoded at all?

That is the rest pagination not the visible view-pagesize which is a component property. It could be extracted to a constant though. Because it should be some static optimal value which would reduce the number of calls with the optimal payload size, without putting stress on rest and decreasing the round-trip; and since 10 txs fits to the visible area usually, 20 seemed reasonable to me?